pixeltable/catalog/globals.py

```diff
@@ -66,8 +68,8 @@ def validated(cls, name: str, error_prefix: str) -> MediaValidation:
         try:
             return cls[name.upper()]
         except KeyError:
-            val_strs = ', '.join(f'{s.lower()!r}' for s in cls.__members__.keys())
-            raise excs.Error(f'{error_prefix} must be one of: [{val_strs}]')
+            val_strs = ', '.join(f'{s.lower()!r}' for s in cls.__members__)
```

Comment:
What's wrong with making keys() explicit?

Reply:
Here's what ruff has to say about it:
https://docs.astral.sh/ruff/rules/in-dict-keys/